testing/variables.tf

```diff
@@ -0,0 +1,73 @@
+variable "aws_region" {
+  type        = string
+  description = "AWS Region the instance is launched in"
+  default     = "us-gov-west-1"
+}
+
+variable "aws_subnet" {
+  description = "List of vectors of subnets and Availability Zones"
+  type        = string
+  default     = "vpc-07402b459d3b18976"
+}
+
+variable "tf_user" {
+  type    = string
+  default = "rke2-ansible-github-actions"
+}
+
+### AMI
+variable "instance_type" {
+  type        = string
+  description = "AWS type of instance"
+  default     = "t2.xlarge"
+}
+
+# OS Options
+#  rhel8
+#  rhel7
+#  opensuse
+#  ubuntu20
```

Comment:
pet peeve, but I'd prefer this to be called something like "ubuntu2004" to distinguish from the ubuntu 20.10 release (Groovy Gorilla)

Reply:
Since rke2 does not technically support ubuntu 20.10 (https://docs.rke2.io/install/requirements/#operating-systems) I left it off.  Technically rke2 supports RHEL 7.8, but I dropped the minor version to just use the var `rhel7`.  Using `ubuntu20` seemed consistent with the other naming conventions.
I can change it though if it makes you feel better.  Granted the `opensuse` one should be `opensuse15` to follow the convention as well. 
Thoughts @capta1nk1rk ?

Reply:
sounds like we need a broader conversation about parameter naming conventions. having said that I agree and I believe we should be more verbose as often as possible, including `opensuse15`

Reply:
I would concur with @capta1nk1rk and @TestInProduction0 ....my expectation would be that the list will most likely grow larger to include a myriad of major / minor versions over time - so having verbosity in OS ID should be accounted for in schema day 1...

Reply:
So should we continue with that and use `rhel78` then instead of `rhel7`?

Reply:
a clean pattern I've seen past is with Github runners ... for example - 
![image](https://user-images.githubusercontent.com/8986826/131378206-be3445b8-d17a-413f-974f-570750606eda.png)

Reply:
are there any concerns about '-' and '.' characters being problematic depending on the env this code is running in? just asking more from an inexperience standpoint @jesseagonzalez

Reply:
@TestInProduction0 I'm hoping this won't be an issue with Terraform. This is the only place this change will happen. I'll test and find out.